spec/controllers/account/page_controller_spec.rb

```diff
@@ -1,8 +1,9 @@
 require 'rails_helper'
+require 'sidekiq/testing'
```

Comment:
could you follow wiki at https://github.com/mperham/sidekiq/wiki/Testing
also, it'd be really helpful if you create a separate sidekiq config for RSpec and include it by metadata, e.g.:
```
RSPec.describe 'SomeWorker', sidekiq: true do
  #.. testing
end
```

Reply:
I can't find how to create a separate config for sidekiq
And where can I set sidekiq option for use ``` sidekiq: true ```